tensorflow/lite/kernels/slice_test.cc

```diff
@@ -238,6 +238,28 @@ TEST_P(SliceOpTest, SliceString) {
                                 "2,0,0,0", "2,0,1,0", "2,0,2,0"}));
 }
 
+TEST(SliceOpTest, SliceInt64) {
+  SliceOpModel<int64_t, int32_t> m({3, 2, 3, 1}, {4}, {4}, TensorType_INT32,
```

Comment:
`tensorflow/lite/kernels/slice_test.cc:242:34: error: no matching constructor for initialization of 'SliceOpModel<int64_t, int32_t>' (aka 'SliceOpModel<long, int>')
  SliceOpModel<int64_t, int32_t>`